app/src/main/java/com/tiptopgoodstudio/androidresources/HomeFragment.java

```diff
@@ -0,0 +1,58 @@
+package com.tiptopgoodstudio.androidresources;
+
+
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.Menu;
+import android.view.MenuInflater;
+import android.view.MenuItem;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.SearchView;
+import android.widget.Toast;
+
+/**
+ * Updated on 03/20/2018 by Olga Agafonova.
+ * The onQueryTextSubmit method is passing the user string to the ResourceListAdapter
+ */
+public class HomeFragment extends Fragment {
+
+    private static final String TAG = "Home";
```

Comment:
We can use the class name as the TAG.  This way if we make any name changes, we don't have to update the tag.
`private static final String TAG = HomeFragment.class.getSimpleName();`

Reply:
Got it, made change in the coming commit.